src/main/resources/static/js/main.js

```diff
@@ -75,26 +75,33 @@ document.addEventListener('DOMContentLoaded', () => {
 
     forms.forEach(item => item.addEventListener('submit', function (e) {
         e.preventDefault();
-
-        let formData = new FormData(this);
-        formData = Object.fromEntries(formData);
-        console.log(formData)
-        ajaxSend(formData);
-        this.reset();
+        var reg = /^([A-Za-z0-9_\-\.])+\@([A-Za-z0-9_\-\.])+\.([A-Za-z]{2,4})$/;
+        var formValue = item.querySelector('.email').value;
+        if (reg.test(formValue) == false) {
+            alert('Введите корректный e-mail');
```

Comment:
Could you please change it to English. I guess English should be default language for us

Reply:
Fixed